Source/Scene/Batched3DModel3DTileContent.js

```diff
@@ -0,0 +1,455 @@
+/*global define*/
+define([
+        '../Core/Color',
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/defineProperties',
+        '../Core/deprecationWarning',
+        '../Core/destroyObject',
+        '../Core/DeveloperError',
+        '../Core/getAbsoluteUri',
+        '../Core/getBaseUri',
+        '../Core/getStringFromTypedArray',
+        '../Core/RequestType',
+        './Cesium3DTileBatchTable',
+        './Cesium3DTileFeature',
+        './Cesium3DTileFeatureTable',
+        './getAttributeOrUniformBySemantic',
+        './Model'
+    ], function(
+        Color,
+        defaultValue,
+        defined,
+        defineProperties,
+        deprecationWarning,
+        destroyObject,
+        DeveloperError,
+        getAbsoluteUri,
+        getBaseUri,
+        getStringFromTypedArray,
+        RequestType,
+        Cesium3DTileBatchTable,
+        Cesium3DTileFeature,
+        Cesium3DTileFeatureTable,
+        getAttributeOrUniformBySemantic,
+        Model) {
+    'use strict';
+
+    /**
+     * Represents the contents of a
+     * {@link https://github.com/AnalyticalGraphicsInc/3d-tiles/blob/master/TileFormats/Batched3DModel/README.md|Batched 3D Model}
+     * tile in a {@link https://github.com/AnalyticalGraphicsInc/3d-tiles/blob/master/README.md|3D Tiles} tileset.
+     *
+     * @alias Batched3DModel3DTileContent
+     * @constructor
+     *
+     * @private
+     */
+    function Batched3DModel3DTileContent(tileset, tile, url, arrayBuffer, byteOffset) {
+        this._tileset = tileset;
+        this._tile = tile;
+        this._url = url;
+        this._model = undefined;
+        this._batchTable = undefined;
+        this._features = undefined;
+
+        /**
+         * Part of the {@link Cesium3DTileContent} interface.
+         */
+        this.featurePropertiesDirty = false;
+
+        initialize(this, arrayBuffer, byteOffset);
+    }
+
+    // This can be overridden for testing purposes
+    Batched3DModel3DTileContent._deprecationWarning = deprecationWarning;
+
+    defineProperties(Batched3DModel3DTileContent.prototype, {
+        /**
+         * Part of the {@link Cesium3DTileContent} interface.
+         */
+        featuresLength : {
+            get : function() {
+                return this._batchTable.featuresLength;
+            }
+        },
+
+        /**
+         * Part of the {@link Cesium3DTileContent} interface.
+         */
+        pointsLength : {
+            get : function() {
+                return 0;
+            }
+        },
+
+        /**
+         * Part of the {@link Cesium3DTileContent} interface.
```

Comment:
You can replace doc everywhere with a one line inherit `/** @inheritdoc */`, which is part of the jsdoc system and if/when we ever wire up inheritance officially, will work the way you would want it to.